src/tests/electron/common/view-controllers/app-controller.ts

```diff
@@ -45,11 +45,19 @@ export class AppController {
         const automatedChecksView = new AutomatedChecksViewController(this.client);
         await automatedChecksView.waitForViewVisible();
 
-        this.setToMinimumSupportedWindowSize();
+        this.resizeWindow();
 
         return automatedChecksView;
     }
 
+    private resizeWindow(): void {
+        let resizeId;
+        this.app.browserWindow.addListener('resize', () => {
```

Comment:
I think there's maybe a misunderstanding here; I don't think this ever actually invokes this.setToMinimumSupportedWindowSize, it's just adding a listener such that it *would* be invoked if some other thing resized the window. I tried running the electron tests locally with this change to confirm my understanding and found that the electron window does not appear to get resized down to minimum with this code in place.